File: 2021/1/day1.py
with open('./day1.txt') as f:
    data = [int(l.strip()) for l in f.readlines()]

part1 = lambda data: sum(data[i] < data[i+1] for i in range(len(data)-1))
part2 = lambda data: sum(data[i] < data[i+3] for i in range(len(data)-3))

print(part1(data))
print(part2(data))

File: 2021/1/day1.jl
data = parse.(Int, readlines(open("day1.txt")))

part1 = sum(data[i-1] < data[i] for i in 2:length(data))
part2 = sum(data[i-3] < data[i] for i in 4:length(data))

println(part1)
println(part2)